firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests/L10nSuite2SnapshotTests.swift

```diff
@@ -166,7 +166,8 @@ class L10nSuite2SnapshotTests: L10nBaseSnapshotTests {
         mozWaitForElementToExist(website.textFields.firstMatch)
         website.textFields.firstMatch.tap()
         website.textFields.firstMatch.typeText("Shoe")
-        website.buttons.element(boundBy: 5).tap()
+        mozWaitForElementToExist(website.otherElements.buttons.firstMatch)
```

Comment:
This tests fails on some locale such as `af`. There may not be multiple matches from Amazon for the specific locale. Now let's just tap on the first match.